Tests/Security/SessionDownloadStrategyTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Tests\Security;
+
+use Sonata\MediaBundle\Security\SessionDownloadStrategy;
+
+/**
+ * Class SessionDownloadStrategyTest.
+ *
+ * @author Ahmet Akbana <ahmetakbana@gmail.com>
+ */
+final class SessionDownloadStrategyTest extends \PHPUnit_Framework_TestCase
+{
```

Comment:
public function testLegacyConstructorUsageStillWorks
{
        $translator = $this->getMock('Symfony\Component\Translation\TranslatorInterface');
        $container = $this->getMock('Something…\ContainerInterface');

        $container->expects($this->once())
            ->method('get')
            ->with('session')
            ->willReturn($this->getMock('Symfony\Component\HttpFoundation\Session\Session'));

        $strategy = new SessionDownloadStrategy($translator, $session, 0);
}